docs/UserManuals/FAQ.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "FAQ"
+sidebar_position: 10
+description: >
+  Frequently Asked Questions
+---
+
+## How to collect data from server with Self-Signed Certificate
```

Comment:
Should it be a question since it's FAQ?

Such as: Failed to collect data from the server with a self-signed certificate.